### ISSUE TITLE
Real-time clock (RTC) 

### ISSUE THREAD

**Issue by cua-cua** [Theseus/theseus#96]
Instead of clock.c file and functions, the RTC clock would be more convenient. (Section 18 from reference manual)

**Comment by Peque** [comment]
@cua-cua Shall we close this? Do you still think the RTC could be useful for us? If so, how/why?

**Comment by cua-cua** [comment]
Yes! Although I guess that when we have operating system this will be carried by it...
